script.js

```diff
@@ -0,0 +1,40 @@
+// FORM SUBMISSION
+// On the form page, display an alert that says "Thank you. 
+// The form information has been received" when the form has been submitted.
+// Console log the values inside the form, you can build a string or build an object.
+
+//creates an empry array
+let formData = [];
+
+//add EventListener to the form
+form.addEventListener('submit', (e) => {
+  e.preventDefault();
+
+  let formValues = {
+    name: document.getElementById('name').value,
+    email: document.getElementById('email').value,
+    address: document.getElementById('address').value,
+    city: document.getElementById('city').value,
+    state: document.getElementById('state').value,
+    zipcode: document.getElementById('zipcode').value,
+    firstAdoptY: document.getElementById('yes').value,
```

Comment:
Be careful with how you treat radio buttons. They have an extra field called checked that you should report instead of value. As it is now your form always reports yes for a first time adopter.